docs/okeps/okep-5672-metadata-service.md

```diff
@@ -0,0 +1,245 @@
+# OKEP-5672: Metadata Service
+
+* Issue: [#5672](https://github.com/ovn-org/ovn-kubernetes/issues/5672)
+
+## Problem Statement
+
+VM cloud platforms typically offer metadata service at 169.254.169.254.
+For Kubevirt VMs running on ovn-kubernetes, we would like to offer the same kind of support.
+
+## Goals
+
+* Allow a user to configure metadata service access with their UDN that enables access to 169.254.169.254 with Internal Traffic Policy (ITP) local
```

Comment:
ITP local is an implementation detail. Might be worth calling it out more generically within the goals section.

Reply:
Yeah its more of a use case thing, I should move it there